### ISSUE TITLE
Long Yun (龍雲) - YUN (Yunnan)

### ISSUE THREAD

**Issue by TheRealVladimirLemon** [Kaiserreich/Kaiserreich-Research-Assistance-Programme#5]
Long Yun is a man with many pictures; however, he has few actually decent sources. The current in-game used sources is from 1928, so it is far from optimal. The only other somewhat passable source we have is from 1945, but is still far worse in quality and very suboptimal as well. We also have a 1944 source with him next to Joseph Stilwell, but his glasses being goofy kind of destroys the photos potential.
![Long_yun_photo](https://github.com/Kaiserreich/Kaiserreich-Research-Assistance-Programme/assets/90009369/0816bc45-f8b6-4267-9c2a-7f376708b343)
![zgse201704zgse20170411-1-l](https://github.com/Kaiserreich/Kaiserreich-Research-Assistance-Programme/assets/90009369/087b2845-23b7-435a-9ad4-047ecb310d5c)
![Screen_Shot_2023-08-03_at_4 39 14_PM](https://github.com/Kaiserreich/Kaiserreich-Research-Assistance-Programme/assets/90009369/853d9033-07e8-4164-90f1-933d1de86d95)
This photo from 1945 is located in Taipei, at the Acedmia Historica Museum, and it can be found on the online repository here https://ahonline.drnh.gov.tw/index.php?act=Archive. This archive has A LOT of poorly scanned but phenomenal sources, and it's public access. If there is anyone who can go here in person, please contact me at therealvladimirlemon on discord. Ping me in the kaiserreich ask_a_dev channel if you are having trouble contacting me directly on discord. This is a potential goldmine for many Chinese leader sources.

Basically anything of decent quality that's larger than the size of a standard HOI4 portrait would be great to have for this. Sometime in the 1930s or before 1945 would be optimal date-wise.

**Comment by Luki1223PL** [comment]
Found two:
1) Unknown year, looks older though
![GENERA~1](https://github.com/Kaiserreich/Kaiserreich-Research-Assistance-Programme/assets/164225798/63d8b9d8-5f00-48d0-b17d-34c46abea262)
https://www.alamy.com/general-long-yun-27-november-1884-27-june-1962-was-governor-and-warlord-of-the-chinese-province-of-yunnan-from-1927-to-october-1945-when-he-was-overthrown-in-a-coup-known-as-the-kunming-incident-by-du-yuming-under-the-order-of-chiang-kai-shek-image482095905.html?imageid=F2465BAF-1691-45FE-B7C0-6255A3164AA1&p=75935&pn=1&searchId=a5a47e30af3db4f2f81b99c2faf842cb&searchtype=0
2) Also unknown, also older
![iunsql4cngx71](https://github.com/Kaiserreich/Kaiserreich-Research-Assistance-Programme/assets/164225798/f294431d-249b-4537-a12a-e0987fea1447)
https://www.reddit.com/r/TNOmod/comments/qm6l5m/i_found_an_actual_pic_of_long_yun_in_his_later/

**Comment by ShandongOpiumCo** [comment]
Here is a source with Long Yun and Chennault from 1944, i'd say it's a little bit more clear but idk
![sdyx_002_005633](https://github.com/Kaiserreich/Kaiserreich-Research-Assistance-Programme/assets/152008632/7a07c2d7-1440-4ceb-a153-681a0d10ac0f)

**Comment by ihategre** [comment]
There's a version of the above image in larger resolution on [here](https://waralbum.ru/332193/).

**Comment by KevinLiu022** [comment]
1. Long Yun with Chiang Kai-shek, 1936

![image](https://github.com/Kaiserreich/Kaiserreich-Research-Assistance-Programme/assets/70558891/c65bbd83-2a62-4a60-9f69-d74eb0809c94)

Source: https://www.theatlantic.com/photo/2011/06/world-war-ii-before-the-war/100089/

2. Long Yun with He Yingqin, 1938

![image](https://github.com/Kaiserreich/Kaiserreich-Research-Assistance-Programme/assets/70558891/f7abcd6b-9b89-4b5d-86e0-659002ae6700)

Source: https://commons.wikimedia.org/wiki/File:Long_Yun_and_He_Yingqin.jpg

3. Long Yun with Chiang Kai-shek, 1930?

![image](https://github.com/Kaiserreich/Kaiserreich-Research-Assistance-Programme/assets/70558891/c3c074f1-d76c-4cef-b6b2-ef0d07646724)

Source: https://commons.wikimedia.org/wiki/File:Long_and_Chiang_2.jpg

**Comment by Xiangwan-damowang** [comment]
The best picture I think
Long Yun with Chennault
![downloadfile_edit_200707671084007](https://github.com/user-attachments/assets/30d65981-6f18-437e-bc7f-b7dd4c842480)

Source:Internet
